x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/rules/types.ts

```diff
@@ -11,18 +11,26 @@ import type { AnalyticsServiceSetup } from '@kbn/core/public';
 import type { SavedObjectsClientContract } from '@kbn/core/server';
 import type { PackageService } from '@kbn/fleet-plugin/server';
 import type { InferenceClient } from '@kbn/inference-plugin/server';
+import type { IndexAdapter, IndexPatternAdapter } from '@kbn/index-adapter';
 import type {
+  RuleMigration,
   RuleMigrationTranslationResult,
   UpdateRuleMigrationData,
 } from '../../../../common/siem_migrations/model/rule_migration.gen';
-import {
-  type RuleMigration,
-  type RuleMigrationResource,
-} from '../../../../common/siem_migrations/model/rule_migration.gen';
+import { type RuleMigrationResource } from '../../../../common/siem_migrations/model/rule_migration.gen';
 import type { RuleVersions } from './data/rule_migrations_data_prebuilt_rules_client';
 
 export type Stored<T extends object> = T & { id: string };
 
+export interface SiemMigration {
+  id: string;
```

Comment:
not needed

Reply:
🤦 .. you are right. Fixed in 60e84507f31